### PR TITLE
Split cell resolve in two

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
### Purpose of this PR

Cell data are made of a few arrays, some of them change between states (like sh data), others do not (like probe validity)
Change resolve cells to not reload shared data when the state changes.

Additionally, it moves probe validity data from the per state data file to the shared one to reduce data duplication.
This will require rebaking existing projects

---
### Testing status

Tested with a baking set containing two states. Verified that closing and reopening the project loaded cells correctly.
Verified that when baking a state, shared data are correctly reloaded and switching works

Updated the runtime test baked data

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs [hunk 1]
```diff
@@ -167,6 +170,7 @@ void OnEnable()
         {
             ProbeReferenceVolume.instance.RegisterPerSceneData(this);
 
+            ResolveSharedCellData();
```
Comment (FrancescoC-unity) [inline com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs:173]: If this is the only place where we do it, what happens with multiple bakes when we queue up the loading?

Comment (adrien-de-tocqueville) [reply com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs:173]: At the end of baking there is already a call to resolvecells, which reloads all cell data

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs:173]: Ok, should be fine worth pinging @Unity-Technologies/gfx-qa-hdrp  to hammer a bit the loading process in testing this PR (loading already baked and doing a bunch of bakes after changing the scene structure a bit)

**Review by FrancescoC-unity — APPROVED** [review]

**Review by TomasKiniulis — APPROVED** [review]
Things tested:
- Loading baked scenes in multiple sets and switching between various states
- Reloading the project and loading scenes again
- Made sure lighting remains correct in Classroom project
- Tested with multiple lighting setups
- Rebaking lighting with slight object changes
- Rebaking lighting causing "Out Of Date" to other states

Looks good! Haven't ran into any PR specific issues.

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**